projects/core/src/lib/backend/backend.model.ts

```diff
@@ -12,15 +12,6 @@ export interface RegistrationDict {
     id: UUID;
 }
 
-export interface SessionDict {
```

Comment:
Do we not need to wrap everything like we do in Python (I mean converting to and from internal types)? Our core is basically also a library.

Or do we view it as an application?

Reply:
Well, there is a session class that wraps it. But we can replace the dict types.